lib/full-site-editing/edit-site-page.php

```diff
@@ -89,19 +89,17 @@ function gutenberg_edit_site_init( $hook ) {
 	 */
 	$current_screen->is_block_editor( true );
 
-	$settings = array_merge(
-		gutenberg_get_default_block_editor_settings(),
-		array(
-			'siteUrl'                              => site_url(),
-			'postsPerPage'                         => get_option( 'posts_per_page' ),
-			'styles'                               => gutenberg_get_editor_styles(),
-			'defaultTemplateTypes'                 => gutenberg_get_indexed_default_template_types(),
-			'defaultTemplatePartAreas'             => gutenberg_get_allowed_template_part_areas(),
-			'__experimentalBlockPatterns'          => WP_Block_Patterns_Registry::get_instance()->get_all_registered(),
-			'__experimentalBlockPatternCategories' => WP_Block_Pattern_Categories_Registry::get_instance()->get_all_registered(),
-		)
+	$custom_settings     = array(
```

Comment:
`$custom_settings` will be merged with `gutenberg_get_default_block_editor_settings` in the `gutenberg_get_block_editor_settings` function, which is why we remove the `array_merge` here.

 https://github.com/WordPress/gutenberg/blob/c96feebc4dafa087b5d247ad9c3a6ace5cb96d3c/lib/compat/wordpress-5.8/block-editor.php#L264-L272